packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -214,7 +214,7 @@ export default class Meetings extends WebexPlugin {
             meeting.locusInfo.initialSetup(data.locus);
           }
         })
-          .finally(() => {
```

Comment:
Create sometime throws an error for the /meetingInfo . Irrespective of that we still send a meeting:added event so we need this in finally